test/plugin/scenarios/httpclient-5.x-scenario/config/expectedData.yaml

```diff
@@ -0,0 +1,120 @@
+                 null, parentService: httpclient-5.x-scenario, traceId: not null}
+            skipAnalysis: 'false'
+      - segmentId: not null
+        spans:
+          - operationName: /httpclient-5.x/back
+            parentSpanId: 0
+            spanId: 1
+            spanLayer: Http
+            startTime: nq 0
+            endTime: nq 0
+            componentId: not null
+            isError: false
+            spanType: Exit
+            peer: 127.0.0.1:8080
+            tags:
+              - {key: url, value: 'http://127.0.0.1:8080/httpclient-5.x/back'}
+              - {key: http.method, value: GET}
+            skipAnalysis: 'false'
+          - operationName: httpasyncclient/local
+            parentSpanId: -1
+            spanId: 0
+            spanLayer: Http
+            startTime: nq 0
+            endTime: nq 0
+            componentId: not null
+            isError: false
+            spanType: Local
+            peer: ''
+            refs:
+              - {parentEndpoint: /httpclient-5.x/case/asyncGet, networkAddress: '',
+                refType: CrossThread, parentSpanId: 0, parentTraceSegmentId: not null, parentServiceInstance: not
+                 null, parentService: httpclient-5.x-scenario, traceId: not null}
+            skipAnalysis: 'false'
+      - segmentId: not null
+        spans:
+          - operationName: /httpclient-5.x/case/asyncGet
+            parentSpanId: -1
+            spanId: 0
+            spanLayer: Http
+            startTime: nq 0
+            endTime: nq 0
+            componentId: not null
+            isError: false
+            spanType: Entry
+            peer: ''
+            tags:
+              - {key: url, value: 'http://127.0.0.1:8080/httpclient-5.x/case/asyncGet'}
+              - {key: http.method, value: GET}
+            refs:
+              - {parentEndpoint: /httpclient-5.x/case/get, networkAddress: '127.0.0.1:8080',
+                refType: CrossProcess, parentSpanId: 1, parentTraceSegmentId: not null, parentServiceInstance: not
+                 null, parentService: httpclient-5.x-scenario, traceId: not null}
+            skipAnalysis: 'false'
+      - segmentId: not null
+        spans:
+          - operationName: /httpclient-5.x/case/asyncGet
+            parentSpanId: 0
+            spanId: 1
+            spanLayer: Http
+            startTime: nq 0
+            endTime: nq 0
+            componentId: not null
+            isError: false
+            spanType: Exit
+            peer: 127.0.0.1:8080
+            tags:
+              - {key: url, value: 'http://127.0.0.1:8080/httpclient-5.x/case/asyncGet'}
+              - {key: http.method, value: GET}
+            skipAnalysis: 'false'
+          - operationName: /httpclient-5.x/case/get
+            parentSpanId: -1
+            spanId: 0
+            spanLayer: Http
+            startTime: nq 0
+            endTime: nq 0
+            componentId: not null
```

Comment:
Please use real componentId in the expectedData file.

Reply:
OK, I add it.